Tests/Features/Page/sitemapXML.feature.wip

```diff
@@ -0,0 +1,35 @@
+@javascript @alice(Page) @reset-schema
+Feature: Generate sitemap.xml
+
+  Background:
+    Given I run the viewreference generation command
+    Given I run the sitemap clear command
+    And I am on the homepage
+
+  Scenario: I go to the sitemap.xml page
+    Given I am on "http://en.victoire.io:8000/app_domain.php/sitemap.xml"
+    Then the response should contain "<urlset xmlns=\"http://www.sitemaps.org/schemas/sitemap/0.9\">"
+    And the response should contain "<loc>http://en.victoire.io:8000/app_domain.php/</loc>"
+    And the response should contain "<loc>http://en.victoire.io:8000/app_domain.php/english-test</loc>"
```

Comment:
The sitemap should be empty after we ran the sitemap clear command.

Reply:
Actually, this is not how it works.
When the sitemap is empty (_cleared or not generated yet_), the `xmlAction` (https://github.com/Victoire/victoire/blob/2.3/Bundle/SitemapBundle/Controller/SitemapController.php#L39) recomputes on the fly the sitemap.

The generation command stands for big sites, with a large number of Business entities, to prepare and cache the sitemap and then avoid a server timeout.